dc/council/code/titles/9/sections/9-1103.02.xml

```diff
@@ -47,13 +47,13 @@
       <num>(c)</num>
       <text>Notwithstanding the provisions of paragraph (a) of this section, this Act shall apply to taxicabs and other vehicles used in performing a bona fide taxicab service having a seating capacity of eight passengers or less in addition to the driver thereof with respect only to (i) the rate or charges for transportation from one signatory to another within the confines of the Metropolitan District, and (ii) requirements for minimum insurance coverage.</text>
     </para>
+    <aftertext>
```

Comment:
Why make it an aftertext for this `<para>` as opposed to the first `<text>` element in the next `<para>`?